testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java

```diff
@@ -227,13 +220,6 @@ protected VMRunnerConfiguration createVMRunner(
     argv.add(RemoteArgs.PROTOCOL);
     argv.add(protocol.toString());
 
-    argv.add(RemoteArgs.VERSION);

```

Comment:
don't pass the '-version' arg, let RemoteTestNG itself detect the TestNG version at runtime